app/shared/ui-foundation/src/desktopMain/kotlin/ui/foundation/theme/AppTheme.desktop.kt

```diff
@@ -31,4 +44,55 @@ actual fun appColorScheme(
             modifyColorSchemeForBlackBackground(colorScheme, isDark, useBlackBackground)
         },
     )
+}
+
+@Composable
+private fun DesktopDynamicTheme(
+    useDynamicTheme: Boolean,
+    useBlackBackground: Boolean,
+    isDark: Boolean,
+): ColorScheme? {
+    return when {
+        !useDynamicTheme -> null
+        LocalPlatform.current.isWindows() -> {
+            val platformWindow = LocalPlatformWindow.current
+            val isDarkState = rememberUpdatedState(isDark)
```

Comment:
```suggestion
            val currentIsDark by rememberUpdatedState(isDark)
```

Reply:
> ```suggestion
>             val currentIsDark by rememberUpdatedState(isDark)
> ```

不能用by，by的话和remember(isDark)的效果一样了

Reply:
那这里为什么要 remember 一个 state 对象？

Reply:
> 那这里为什么要 remember 一个 state 对象？

State变化时要重新生成一个flow，flow只检测state的value变化